scripts/sendData.ts

```diff
@@ -0,0 +1,36 @@
+import '@nomiclabs/hardhat-ethers'
+import { ethers } from 'hardhat'
+import { BigNumber } from 'ethers'
+
+async function main() {
+  const [signer] = await ethers.getSigners()
+
+  // Replace with your deployed contract address
+  const contractAddress = '0xbA6E2502B4Ef771582bD4517182440cC6D31c8e7'
```

Comment:
After deploying the dummy contract replace this

Reply:
Why don't we just have a flag that optionally deploys the contract for you, and then use the address in the same script, We could just have it do that based on a env var or a cmd flag.